uk.co.compucorp.civicrm.tasksassignments/scss/civitasks/partials/_nav-tabs.scss

```diff
@@ -0,0 +1,34 @@
+// Variables
+$inactive-tab:    #F4F7F8;
```

Comment:
how about `inactive-tab-light-blue-color` or `light-gray-color` that can be used at multiple places and not just in active tab and similarly for others